### DISCUSSION TITLE
\mathchardef Not Supported in Equation Preview

### DISCUSSION THREAD

**Discussion by Squareless-XD** [James-Yu/LaTeX-Workshop#3884] [Q&A]
\mathchardef\mhyphen="2D
proof $\mathrm{ABC\mhyphen DEF}$

When using \mathchardef like this, the hovering preview won't work properly
![image](https://github.com/James-Yu/LaTeX-Workshop/assets/85856989/09a75782-b453-4575-9f29-7593e002bcd9)
It is probably not supported. Wish it could be fixed.

**Comment by James-Yu** [comment]
You may report to MathJaX.
